ci/playbooks/tcib/run.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
Rename the file from run.yaml to run.yml

Reply:
done